Source/Core/Core/HW/WiiSaveStructs.h

```diff
@@ -101,7 +101,8 @@ class Storage
   };
 
   virtual ~Storage() = default;
-  virtual bool SaveExists() { return true; }
+  virtual bool SaveExists() = 0;
```

Comment:
Given this code is being modified, would it make sense to make this const qualified?